### PR TITLE
feat: support LiveSync to Wi-Fi devices

### PR DESCRIPTION
**Pull request by DimitarTachev**
## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] You have signed the [CLA](http://www.nativescript.org/cla).
- [x] All existing tests are passing: https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base
- [ ] Tests for the changes are included.

Related to: https://github.com/NativeScript/nativescript-cli/issues/4966

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
test cli-device cli-templates cli-run cli-debug cli-test cli-misc

**Comment by Fatme** [comment]
test cli-device cli-templates cli-run cli-debug cli-test cli-misc

**Comment by Fatme** [comment]
Update `ios-device-lib` after publishing it to npm.

**Comment by Fatme** [comment]
test cli-device cli-templates cli-run cli-debug cli-test cli-misc

File: lib/device-sockets/ios/socket-request-executor.ts [hunk 1]
```diff
@@ -6,22 +6,39 @@ export class IOSSocketRequestExecutor implements IiOSSocketRequestExecutor {
...
+		const mainRequestName = this.$iOSNotification.getAttachRequest(appId, deviceId);
+		const readyRequestName = this.$iOSNotification.getReadyForAttach(appId);
+		await this.executeRequest(mainRequestName, readyRequestName, appId, deviceId, timeout);
+	}
+
+	public async executeRefreshRequest(device: Mobile.IiOSDevice, appId: string): Promise<boolean> {
+		const deviceId = device.deviceInfo.identifier;
+		const mainRequestName = this.$iOSNotification.getRefreshRequest(appId);
+		const readyRequestName = this.$iOSNotification.getAppRefreshStarted(appId);
+		const timeout = 5;
```
Comment (DimitarTachev) [inline lib/device-sockets/ios/socket-request-executor.ts]: We could accept this as an argument

File: lib/device-sockets/ios/socket-request-executor.ts [hunk 2]
```diff
@@ -6,22 +6,39 @@ export class IOSSocketRequestExecutor implements IiOSSocketRequestExecutor {
...
+		const deviceId = device.deviceInfo.identifier;
+		const mainRequestName = this.$iOSNotification.getAttachRequest(appId, deviceId);
+		const readyRequestName = this.$iOSNotification.getReadyForAttach(appId);
+		await this.executeRequest(mainRequestName, readyRequestName, appId, deviceId, timeout);
+	}
+
+	public async executeRefreshRequest(device: Mobile.IiOSDevice, appId: string): Promise<boolean> {
+		const deviceId = device.deviceInfo.identifier;
+		const mainRequestName = this.$iOSNotification.getRefreshRequest(appId);
+		const readyRequestName = this.$iOSNotification.getAppRefreshStarted(appId);
```
Comment (DimitarTachev) [inline lib/device-sockets/ios/socket-request-executor.ts]: readyRequestName -> refreshRequestStartedName

File: lib/device-sockets/ios/socket-request-executor.ts [hunk 3]
```diff
@@ -6,22 +6,39 @@ export class IOSSocketRequestExecutor implements IiOSSocketRequestExecutor {
...
+		const timeout = 5;
+
+		const result = await this.executeRequest(mainRequestName, readyRequestName, appId, deviceId, timeout);
+
+		return result;
+	}
+
+	private async executeRequest(mainRequestName: string, readyRequestName: string, appId: string, deviceId: string, timeout: number): Promise<boolean> {
```
Comment (DimitarTachev) [inline lib/device-sockets/ios/socket-request-executor.ts]: readyRequestName -> successfulyExecutedNotificationName

**Comment by Fatme** [comment]
test cli-device cli-templates cli-run cli-debug cli-test cli-misc

**Comment by Fatme** [comment]
test cli-device cli-templates cli-run cli-debug

**Comment by Fatme** [comment]
test cli-device cli-templates cli-run cli-debug

**Comment by Fatme** [comment]
test cli-templates

**Comment by Fatme** [comment]
test cli-device cli-run cli-debug

**Comment by Fatme** [comment]
test cli-device cli-templates cli-run cli-debug cli-test cli-misc

**Comment by Fatme** [comment]
test cli-device cli-templates cli-run cli-debug cli-test cli-misc

**Pull request merged**